Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -87,13 +87,15 @@
         'Geom' : 'DB:Extended',
         'GT' : 'auto:phase1_2017_realistic',
         'HLTmenu': '@relval2016',
+        'Custom': 'HLTrigger/Configuration/customizeHLTTrackingForPhaseI2017.customizeHLTPhaseIPixelGeom',
```

Comment:
AFAICT the customize gets now called for each of the workflow steps (not just HLT), but in practice the customize should have no effect on the other steps (in RECO `ClusterShapeHitFilterESProducer` `PixelShapeFile` already points to the "new" shape file).

Thought to note anyway.